index.d.ts

```diff
@@ -1,15 +1,20 @@
 import { Component, CSSProperties } from 'react';
 import { Middleware, Reducer, Action } from 'redux';
 
+export const DEFAULT_SCOPE: string;
+export const HIDE: string;
+export const SHOW: string;
+export const RESET: string;
+
 export interface LoadingBarContainerProps {
-  scope?: string
-  style?: CSSProperties;
   className?: string;
-  actions?: Object;
```

Comment:
This did not look to be defined on the container, not sure when it was removed.

Reply:
It was probably added by mistake in the first place. Thank you for cleaning those up!